databroker/src/glob.rs

```diff
@@ -136,919 +159,919 @@ mod tests {
-        "Vehicle.Powertrain.TractionBattery.Temperature.Average",
-        "Vehicle.Powertrain.TractionBattery.Temperature.Max",
-        "Vehicle.Powertrain.TractionBattery.Temperature.Min",
-        "Vehicle.Powertrain.Transmission.ClutchEngagement",
-        "Vehicle.Powertrain.Transmission.ClutchWear",
-        "Vehicle.Powertrain.Transmission.CurrentGear",
-        "Vehicle.Powertrain.Transmission.DiffLockFrontEngagement",
-        "Vehicle.Powertrain.Transmission.DiffLockRearEngagement",
-        "Vehicle.Powertrain.Transmission.DriveType",
-        "Vehicle.Powertrain.Transmission.GearChangeMode",
-        "Vehicle.Powertrain.Transmission.GearCount",
-        "Vehicle.Powertrain.Transmission.IsElectricalPowertrainEngaged",
-        "Vehicle.Powertrain.Transmission.IsLowRangeEngaged",
-        "Vehicle.Powertrain.Transmission.IsParkLockEngaged",
-        "Vehicle.Powertrain.Transmission.PerformanceMode",
-        "Vehicle.Powertrain.Transmission.SelectedGear",
-        "Vehicle.Powertrain.Transmission.Temperature",
-        "Vehicle.Powertrain.Transmission.TorqueDistribution",
-        "Vehicle.Powertrain.Transmission.TravelledDistance",
-        "Vehicle.Powertrain.Transmission.Type",
-        "Vehicle.Powertrain.Type",
-        "Vehicle.RoofLoad",
-        "Vehicle.Service.DistanceToService",
-        "Vehicle.Service.IsServiceDue",
-        "Vehicle.Service.TimeToService",
-        "Vehicle.Speed",
-        "Vehicle.StartTime",
-        "Vehicle.Trailer.IsConnected",
-        "Vehicle.TraveledDistance",
-        "Vehicle.TraveledDistanceSinceStart",
-        "Vehicle.TripDuration",
-        "Vehicle.TripMeterReading",
-        "Vehicle.VehicleIdentification.AcrissCode",
-        "Vehicle.VehicleIdentification.BodyType",
-        "Vehicle.VehicleIdentification.Brand",
-        "Vehicle.VehicleIdentification.DateVehicleFirstRegistered",
-        "Vehicle.VehicleIdentification.KnownVehicleDamages",
-        "Vehicle.VehicleIdentification.MeetsEmissionStandard",
-        "Vehicle.VehicleIdentification.Model",
-        "Vehicle.VehicleIdentification.OptionalExtras",
-        "Vehicle.VehicleIdentification.ProductionDate",
-        "Vehicle.VehicleIdentification.PurchaseDate",
-        "Vehicle.VehicleIdentification.VIN",
-        "Vehicle.VehicleIdentification.VehicleConfiguration",
-        "Vehicle.VehicleIdentification.VehicleInteriorColor",
-        "Vehicle.VehicleIdentification.VehicleInteriorType",
-        "Vehicle.VehicleIdentification.VehicleModelDate",
-        "Vehicle.VehicleIdentification.VehicleSeatingCapacity",
-        "Vehicle.VehicleIdentification.VehicleSpecialUsage",
-        "Vehicle.VehicleIdentification.WMI",
-        "Vehicle.VehicleIdentification.Year",
-        "Vehicle.VersionVSS.Label",
-        "Vehicle.VersionVSS.Major",
-        "Vehicle.VersionVSS.Minor",
-        "Vehicle.VersionVSS.Patch",
-        "Vehicle.Width",
+        "Vehicle/_kuksa/databroker/GitVersion",
```

Comment:
We are still using the regex matching (for tokens), so the  tests for that should not be removed. Or in other words, the tests should be removed when the code they are testing is removed.

Reply:
Will add them back